docs/advanced/input_files/input-main.md

```diff
@@ -2775,61 +2775,81 @@ These variables are used to control berry phase and wannier90 interface paramete
 ### nurse
 
 - **Type**: Boolean
-- **Description**: If set to 1, the Hamiltonian matrix and S matrix in each iteration will be written in output.
+- **Description**: Specify whether to output the Hamiltonian matrix and S matrix in each iteration.
```

Comment:
On my personality, this parameter makes no sense, please delete this parameter in doc and code.